modules/caddyhttp/templates/templates.go

```diff
@@ -64,6 +64,16 @@ func init() {
 // {{env "VAR_NAME"}}
 // ```
 //
+// ##### `placeholder`
+//
+// Gets an [placerholder variable](/docs/conventions#placeholders). Shorthands are not supported at the moment.
```

Comment:
```suggestion
// Gets an [placeholder variable](/docs/conventions#placeholders).
```

Shorthand placeholders are a Caddyfile-only concept, doesn't make sense to document them in the JSON doc.